locale/templates/auth.ftl

```diff
@@ -215,6 +218,16 @@ recovery-title = Need to reset your password?
 recovery-description = Click the button within the next hour to create a new password. The request came from the following device:
 recovery-action = Create new password
 
+#  Variables:
+#  $productName (String) - The name of the subscribed product, e.g. Mozilla VPN
+subscriptionAccountDeletion-subject = Your { $productName } subscription has been cancelled
+subscriptionAccountDeletion-title = Sorry to see you go
+#  Variables:
+#  $productName (String) - The name of the subscribed product, e.g. Mozilla VPN
+#  $invoiceTotal (String) - The amount of the subscription invoice, including currency, e.g. $10.00
+#  $invoiceDateOnly (String) - The date of the next invoice, e.g. 01/20/2016
+subscriptionAccountDeletion-content-cancelled = You recently deleted your Firefox Account. As a result, we've cancelled your { $productName } subscription. Your final payment of { $invoiceTotal } was paid on { $invoiceDateOnly }.
```

Comment:
Just noticed `we've` needs a curly quote. Will fix.

Reply:
`Firefox Account` needs to be in a placeable